ocaml/xapi/storage_migrate.ml

```diff
@@ -670,7 +670,12 @@ let receive_start ~dbg ~sr ~vdi_info ~id ~similar =
 			| Some vdi ->
 				debug "Cloning VDI %s" vdi.vdi;
 				let vdi = add_to_sm_config vdi "base_mirror" id in
-				Local.VDI.clone ~dbg ~sr ~vdi_info:vdi
+				let c_vdi = Local.VDI.clone ~dbg ~sr ~vdi_info:vdi in
```

Comment:
I like to suggest to use the following names for clarity:

* `clone` for `c_vdi` 
* `clone_size` or `size` for `c_size` 

The same applies for the other change below. Since the code in the two changes is quite similar I would have loved to factor it out but it is difficult because it uses different modules - so I'm fine with keeping it.